be/src/service/internal_service.cpp

```diff
@@ -295,14 +295,21 @@ Status PInternalServiceImplBase<T>::_exec_plan_fragment(brpc::Controller* cntl)
         uint32_t len = ser_request.size();
         RETURN_IF_ERROR(deserialize_thrift_msg(buf, &len, TProtocolType::BINARY, &t_request));
     }
+
     if (UNLIKELY(!t_request.query_options.__isset.batch_size)) {
         return Status::InvalidArgument("batch_size is not set");
     }
-    auto batch_size = t_request.query_options.batch_size;
-    if (UNLIKELY(batch_size <= 0 || batch_size > MAX_CHUNK_SIZE)) {
-        return Status::InvalidArgument(
-                fmt::format("batch_size is out of range, it must be in the range (0, {}], current value is [{}]",
-                            MAX_CHUNK_SIZE, batch_size));
+    bool need_check_chunk_size = true;
+    if (t_request.query_options.__isset.query_type && t_request.query_options.query_type == TQueryType::LOAD) {
+        need_check_chunk_size = false;
```

Comment:
It's to better to add some description here.

Reply:
> It's better to add some description here.

fixed